pokete.py

```diff
@@ -593,25 +617,27 @@ def set_args(self, si):
         """Processes data from save file"""
         self.name = si["user"]
         self.pokes = [Poke((si["pokes"][poke]["name"]
-                                if type(poke) is int else poke),
-                            si["pokes"][poke]["xp"], si["pokes"][poke]["hp"],
-                            shiny = (False
-                                    if "shiny" not in si["pokes"][poke]
-                                    else si["pokes"][poke]["shiny"]),
-                            _attacks = (si["pokes"][poke]["attacks"]
-                                        if "attacks" in si["pokes"][poke]
-                                        else None)
-                            )
-                    for poke in si["pokes"]]
+                            if type(poke) is int else poke),
+                           si["pokes"][poke]["xp"], si["pokes"][poke]["hp"],
+                           shiny=(False
+                                  if "shiny" not in si["pokes"][poke]
+                                  else si["pokes"][poke]["shiny"]),
+                           _attacks=(si["pokes"][poke]["attacks"]
+                                     if "attacks" in si["pokes"][poke]
+                                     else None)
+                           )
+                      for poke in si["pokes"]]
         for j, poke in enumerate(self.pokes):
             poke.set_ap(si["pokes"][j]["ap"])
             if "effects" in si["pokes"][j]:
                 for e in si["pokes"][j]["effects"]:
                     poke.effects.append(getattr(effects, e)(poke))
         try:
-            if si["map"] in ["centermap", "shopmap"]:  # Looking if figure would be in centermap, so the player may spawn out of the center
+            # Looking if figure would be in centermap, so the player may spawn out of the center
+            if si["map"] in ["centermap",
```

Comment:
Why does the line wrap here.

Reply:
The comment would make the line longer than 120 characters, which raises a PEP8 warning E501

Reply:
i see, why you sperated the comment, but not why `if si["map"] in ["centermap", "shopmap"]:` wraps.